web/admin.py

```
@@ -7,6 +7,9 @@
 from django.http import HttpResponseRedirect
 from django.urls import path, reverse
 from django.utils.html import format_html
+from django.db.models import Count, QuerySet
```

Ruff (0.8.2)

`django.db.models.Count` imported but unused

Remove unused import: `django.db.models.Count`

(F401)